src/js/actions/LibraryActions.js

```diff
@@ -71,6 +72,10 @@ export default {
             properties: ['openDirectory', 'multiSelections']
         }, (folders) => {
             if(folders !== undefined) {
+
+                folders.forEach((folder, index, array) => {
+                    array[index] = fs.realpathSync(folder);
```

Comment:
@YurySolovyov  I don't see how you can get rid of synchronous realpath  loop check here